README.md

```diff
@@ -2,7 +2,7 @@
 
 ## A *Caffeine* clone for OS X Yosemite (and Dark Mode). ##
 
-*KeepingYouAwake* is a small menu bar utility for OS X Yosemite and above that can be activated to prevent your Mac from going into sleep.
+*KeepingYouAwake* is a small menu bar utility for OS X Yosemite (10.10) and above that can be activated to prevent your Mac from going into sleep.
```

Comment:
Thanks for the PR! You're right this part of the README is kind of outdated, Yosemite is not that new anymore. What do you think about dropping the codename altogether?

> …a small menu bar utility for OS X (10.10 and newer) that can be…